src/components/Modal.astro

```diff
@@ -104,16 +105,15 @@ const { frontmatter: person, Content } = personMarkdown;
   main {
     display: flex;
     gap: 1rem;
-    overflow: scroll;
+    overflow: auto;
   }
 
   p {
     margin: 0;
   }
 
   .photo {
-    width: 300px;
-    height: 300px;
+    height: 260px;
```

Comment:
I think width is also needed here for when flex direction is horizontal, can you add a quick video with before / after to know where to look?

Reply:
reverted this